interface/main/messages/messages.php

```diff
@@ -1,668 +1,885 @@
 <?php
 /**
  * Message and Reminder Center UI
- *
  * 2013/02/08 Minor tweaks by EMR Direct to allow integration with Direct messaging
  * 2013-03-27 by sunsetsystems: Fixed some weirdness with assigning a message recipient,
- *   and allowing a message to be closed with a new note appended and no recipient.
- *
- * Copyright (c) 2010 OpenEMR Support LLC
- *
- * LICENSE: This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 3
- * of the License, or (at your option) any later version.
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
- *
- * @package OpenEMR
- * @author OpenEMR Support LLC
- * @author Roberto Vasquez <robertogagliotta@gmail.com>
- * @author Rod Roark <rod@sunsetsystems.com>
- * @author Brady Miller <brady.g.miller@gmail.com>
+ * and allowing a message to be closed with a new note appended and no recipient.
+ * @Package OpenEMR
  * @link http://www.open-emr.org
+ * @author OpenEMR Support LLC
+ * @author Roberto Vasquez robertogagliotta@gmail.com
+ * @author Rod Roark rod@sunsetsystems.com
+ * @author Brady Miller brady.g.miller@gmail.com
+ * @author Ray Magauran magauran@medfetch.com
+ * @copyright Copyright (c) 2010 OpenEMR Support LLC
+ * @copyright Copyright (c) 2017 MedEXBank.com
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-
-
 require_once("../../globals.php");
 require_once("$srcdir/pnotes.inc");
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/acl.inc");
 require_once("$srcdir/log.inc");
 require_once("$srcdir/options.inc.php");
+require_once("$srcdir/classes/Document.class.php");
 require_once("$srcdir/gprelations.inc.php");
+require_once "$srcdir/user.inc";
+require_once("$srcdir/MedEx/API.php");
```

Comment:
TODO
Will consider autoloading, although will likely not do this until after it is in the codebase.